### ISSUE TITLE
Error text not included in link text

### ISSUE THREAD

**Issue by ayesha-77** [stringyland/bookmarklets#2]
**Issue description**
* When a heading has an error, error text appears after the relevant heading link. This error text is not included in the link text so it does not announce with the link text when a screen reader user is tabbing to the links. 

**Severity**
* Medium

**Occurs on**
* Chrome 116.0.58/JAWS 2022

**How to replicate:**
1. Navigate to https://stringyland.github.io/bookmarklets/headings-skipped.html and use the bookmarklet
2. Tab to the heading links that have errors.
3. Notice how the error (skipped level) does not announce with the link text.

**Expected behaviour**
* It potentially would be good to include the error text within the link text so that screen reader users can hear it all in one go if they are tabbing. 

**Actual behaviour**
* The error text is not part of the link so users have to switch to using arrow keys to hear whether the heading link is incorrectly marked up.

<img width="960" alt="2023-09-08_15-16-48" src="https://github.com/stringyland/bookmarklets/assets/144410097/08ca9148-f38e-4724-9990-b3e184523506">

**Comment by stringyland** [comment]
Updated with a fix. Moved error text inside `<span class="text">` of each link.
